Source/Core/DiscIO/VolumeCreator.cpp

```diff
@@ -217,8 +215,6 @@ EDiscType GetDiscType(IBlobReader& _rReader)
 	u32 GCMagic = Reader.Read32(0x1C);
 
 	// check for Wii
-	if (WiiMagic == 0x5D1C9EA3 && WiiContainerMagic != 0)
-		return DISC_TYPE_WII;
 	if (WiiMagic == 0x5D1C9EA3 && WiiContainerMagic == 0)
```

Comment:
Isn't checking for WiiContainerMagic unnecessary now? It doesn't feel right to return DISC_TYPE_UNK just because it's set to a non-zero value,